services/ui-src/src/components/report/ReportPageWrapper.tsx

```diff
@@ -69,8 +84,42 @@ export const ReportPageWrapper = () => {
     setCurrentPageId(parentPage.childPageIds[newPageIndex]);
   };
 
+  const saveStatusText = "Last saved " + lastSavedTime;
+
   return (
     <FormProvider {...methods}>
+      <Flex sx={sx.subnavBar}>
+        <Container sx={sx.subnavContainer}>
+          <Flex sx={sx.subnavFlex}>
+            <Flex>
+              <Text sx={sx.submissionNameText}>
+                {report?.state + " QMS Report"}
```

Comment:
Might want to move this string to a dict lookup by report type for future proofing